### ISSUE TITLE
DILATE runs into NaN issue with Taxi dataset

### ISSUE THREAD

**Issue by pratham16cse** [pratham16cse/AggForecaster#8]
On taxi dataset, at step 31 of the training epoch, loss.backward() assigns NaN gradients to all the parameters in the network
